src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Cross-compiling is fine for now. And I verified that despite the PackageTargetFallback, netcoreapp2.0 will get the netstandard2.0 library not net461.